profondo/cnn.py

```diff
@@ -103,7 +202,7 @@ def load_model(x_train, y_train):
 
 def report_callback(epoch, model, x_val, y_val):
     if epoch % 5 == 0:
-        prediction_proba = model.predict_proba(x_val)
+        prediction_proba = model.predict(x_val)
```

Comment:
`predict_proba` isn't defined for `Model` (see [here](https://keras.io/models/model/)) and it's just a legacy wrapper for `predict` in `Sequential` ([implementation](https://github.com/fchollet/keras/blob/master/keras/models.py#L982)).